src/terra-dev-site/app/Application.1.app.mdx

```diff
@@ -0,0 +1,22 @@
+# terra-application
```

Comment:
after looking at https://terra-ui-dep-proposed-r-1xorq5.herokuapp.com/application/terra-application/application, It might look better going with

# Terra Application

Reply:
Also, I think it would be nice if we could add in the badge so it's easy to see what version is current